mobile/native/src/ln_dlc/node.rs

```diff
@@ -189,6 +192,37 @@ impl Node {
             }
         };
 
+        // todo(holzeis): It would be nice if dlc messages are also propagated via events, so the
+        // receiver can decide what events to process and we can skip this component specific logic
+        // here.
+        if let Message::Channel(channel_message) = msg.clone() {
```

Comment:
🔧 I think we can avoid this potentially very expensive `clone` by only cloning the bits that are needed.